### PR TITLE
feat(tutorial): complete step 2 Banglore

### PR DESCRIPTION
**Pull request by SrilekhaMN**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/content/LandingPage/_mixins.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+@mixin landing-page-background() {
+  background-color: Gray;
```
[the remark this answers is not in the document]
Comment (SrilekhaMN) [reply src/content/LandingPage/_mixins.scss]: done

File: src/content/LandingPage/_landing-page.scss [hunk 2]
```diff
@@ -0,0 +1,45 @@
+@import './mixins.scss';
+@import './overrides.scss';
+
+.landing-page__illo {
+  max-width: 100%;
+}
+.landing-page__banner {
+  padding-top: 16px;
+  padding-bottom: 128px;
+  @include landing-page-background;
```
[the remark this answers is not in the document]
Comment (SrilekhaMN) [reply src/content/LandingPage/_landing-page.scss]: done

File: src/content/LandingPage/_mixins.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (kellychurchill) [reply src/content/LandingPage/_mixins.scss]: This is still showing   `background-color: Gray;`

[the remark this answers is not in the document]
Comment (SrilekhaMN) [reply src/content/LandingPage/_mixins.scss]: done

**Pull request closed without merging**